bin/select_stream_targets

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from __future__ import print_function, division
```

Comment:
FYI this boilerplate is no longer necessary; that was for making python 2 act a bit more like python 3 during the 2 -> 3 transition.

Reply:
Agreed and removed.